daal4py/sklearn/utils/fixes.py

```diff
@@ -0,0 +1,19 @@
+from functools import update_wrapper
```

Comment:
Copyright + I don't really understand why you need this file and functions? Why not take these functions from Scikit-learn?